glusterd2/volgen/volgen.go

```diff
@@ -80,6 +80,21 @@ func DeleteClientVolfile(vol *volume.Volinfo) error {
 	return nil
 }
 
+// DeleteClientVoloption deletes the client vol option
+func DeleteClientVoloption(vol *volume.Volinfo) error {
```

Comment:
Not required with volgen2

Reply:
@aravindavk worked on the suggested changes. PR is ready for review.
thanks :)